sodiumpp/include/sodiumpp/locked_string.h

```diff
@@ -37,8 +37,8 @@ class locked_string final
 			 * @return locked_string UNSAFE!
 			 * @{
 			 */
-			static locked_string unsafe_create(const std::string &str);
-			static locked_string unsafe_create(const char *c_str);
```

Comment:
hmm not used? I thought we can use it for tests. If not used then why not remove it indeed